server/src/test/java/io/druid/segment/realtime/appenderator/AppenderatorTest.java

```diff
@@ -144,6 +144,81 @@ public SegmentIdentifier apply(DataSegment input)
     }
   }
 
+  @Test
+  public void testMaxBytesInMemory() throws Exception
```

Comment:
This test is good, but please add another one verifying that the limit is applied across more than one sink in the same appenderator.